ext/payflowpro/info.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<extension key="payflowpro" type="module">
+  <file>payflowpro</file>
+  <name>FIXME</name>
```

Comment:
Again, can we remove all FIXME from info.xml. This looks particularly bad because it is visible in CiviCRM UI.

Do you want mgmt:hidden tag? Maybe not if this is a "complete" extension that can be disabled/not enable by default.

Reply:
PayPal PayFlo Pro Integration